### PR TITLE
IHT-5145 - Address all scalastyle warnings

### PR DESCRIPTION
**Pull request by jennygrahamjones**

### CODE REVIEW

File: app/connectors/securestorage/SecureStorage.scala [hunk 1]
```diff
@@ -151,5 +151,6 @@ trait SecureStorage {
   def objectStorage[T](clazz : Class[T])(implicit f : Format[T], ec : ExecutionContext) =
     new SecureStorageObjectInterface[T](this, clazz)
 
+  //scalastyle:off method.name
```
Comment (grantrobson) [inline app/connectors/securestorage/SecureStorage.scala]: I think you need to switch this back on after you've switched it off. Or use scalastyle:ignore at the end of the line applicable.

File: app/connectors/securestorage/SecureStorageTypedActor.scala [hunk 2]
```diff
@@ -92,6 +92,7 @@ class SecureStorageTypedActor(
       firstMatchOnly=false)
   }
 
+  //scalastyle:off method.name
```
Comment (grantrobson) [inline app/connectors/securestorage/SecureStorageTypedActor.scala]: See previous comment

File: app/constants/Constants.scala [hunk 3]
```diff
@@ -20,6 +20,7 @@ import scala.collection.immutable.ListMap
 /**
  * Created by xavierzanatta on 3/25/15.
  */
+//scalastyle:off magic.number
```
Comment (grantrobson) [inline app/constants/Constants.scala]: See previous comment

File: app/connectors/securestorage/SecureStorage.scala (hunk 1 above)
Comment (jennygrahamjones) [reply app/connectors/securestorage/SecureStorage.scala]: I've switched it back on after the offending method

File: app/connectors/securestorage/SecureStorageTypedActor.scala (hunk 2 above)
Comment (jennygrahamjones) [reply app/connectors/securestorage/SecureStorageTypedActor.scala]: I've switched it back on after the offending method

File: app/constants/Constants.scala (hunk 3 above)
Comment (jennygrahamjones) [reply app/constants/Constants.scala]: I've switched it back on at the end of the object

**Pull request merged**